src/cim/components/modals/CimConfiguration/CimConfigProgressAlert.tsx

```diff
@@ -0,0 +1,114 @@
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+import {
+  Alert,
+  AlertVariant,
+  AlertActionLink,
+  AlertActionCloseButton,
+} from '@patternfly/react-core';
+
+import { useTranslation } from '../../../../common/hooks/use-translation-wrapper';
+import { AgentServiceConfigConditionType } from '../../../types';
+import { CimConfigProgressAlertProps } from './types';
+import { getConditionsByType } from '../../../utils';
+import { isCIMConfigProgressing, isCIMConfigured } from './utils';
+
+const LOCAL_STORAGE_ID_SUCCESS = 'cim-config-progress-alert';
+
+export const resetCimConfigProgressAlertSuccessStatus = () => {
+  window.localStorage.setItem(LOCAL_STORAGE_ID_SUCCESS, '');
+};
+
+export const CimConfigProgressAlert: React.FC<CimConfigProgressAlertProps> = ({
+  agentServiceConfig,
+  assistedServiceDeploymentUrl,
+}) => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const [hideSuccess, setHideSuccess] = React.useState(true);
+
+  React.useEffect(
+    () => {
+      setHideSuccess(window.localStorage.getItem(LOCAL_STORAGE_ID_SUCCESS) === 'closed');
```

Comment:
why is this done via local storage ? Wouldnt it be better to use React context ?

Reply:
To have it permanent.
The related Alert is permanently dismissed till next "Configure" action.